Tests/SnapshotTests.swift

```diff
@@ -0,0 +1,66 @@
+// Created by Cal Stephens on 12/8/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+@testable import Lottie
+import SnapshotTesting
+import XCTest
+
+// MARK: - SnapshotTests
+
+class SnapshotTests: XCTestCase {
+
+  /// Snapshots all of the sample animation JSON files visible to this test target
+  func testLottieSnapshots() {
+    #if !os(iOS)
+    // We only run snapshot tests on iOS, since running snapshot tests
+    // for macOS and tvOS would triple the number of snapshot images
+    // we have to check in to the repo.
+    XCTFail("Snapshot tests should only be ran on iOS")
```

Comment:
If we'd like to prevent this function from running we should be able to throw an error which will both fail the test and prevent the below from running

Reply:
aha good point, that sounds like a better approach